src/olympia/addons/serializers.py

```diff
@@ -300,7 +300,7 @@ def fake_version_object(self, obj, data):
             version = Version(
                 addon=obj, id=data['id'],
                 reviewed=self.handle_date(data['reviewed']),
-                version=data['version'])
+                version=data['version'], channel=data['channel'])
```

Comment:
I don't think we need this, at least not right now. `channel` is only exposed in the full `VersionSerializer`, which is only used by `AddonVersionViewSet`, which doesn't use elasticsearch. So this change and the one in the indexer shouldn't be needed.

*If* we really want the channel in ES though, we'd need the following:
- In the indexer, also change the mapping. See `version_mapping`.
- In the serializer, use `data.get('channel')`, possibly with a fallback (otherwise it'd break everything using the serializer until the next reindex that introduces the new field to the mapping)
- Tests :)